wgpu-rs/src/lib.rs

```diff
@@ -276,7 +276,9 @@ where
     T: Copy,
 {
     pub fn fill_from_slice(self, slice: &[T]) -> Buffer {
-        self.data.copy_from_slice(slice);
+        if slice.len() != 0 {
+            self.data.copy_from_slice(slice);
```

Comment:
is `copy_from_slice` not valid for slices of size 0 ?

Reply:
This was needed for my initial implementation, but not anymore. Much nicer now.